mods/hv/sequences/buildings.yaml

```
@@ -2284,6 +2288,18 @@ orepurifier:
 		ZOffset: 512
 		Offset: 0, -2
 	idle:
+		ShadowStart: 1
+		Length: 1
+		Combine:
+			0:
+				Filename: bits/orepurifier.png
+				Length: 1
+			1:
+				Filename: bits/orepurifier-shadow.png
+				Length: 1
+				ZOffset: 1768
+				Offset: 13,24
+	idle-animation:
```

```suggestion
	active:
```

---

the default is `idle-overlay`:

```suggestion
	active-overlay:
```